proto/substrait/algebra.proto

```diff
@@ -115,6 +115,7 @@ message ReadRel {
       enum FileFormat {
         FILE_FORMAT_UNSPECIFIED = 0;
         FILE_FORMAT_PARQUET = 1;
+        FILE_FORMAT_PARQUET = 2;
```

Comment:
```suggestion
        FILE_FORMAT_ORC = 2;
```

Reply:
Thanks for your review. I have updated it to `FILE_FORMAT_ORC`.